R/expr-s3-base.R

```diff
@@ -1,3 +1,31 @@
+#' @export
+`$.polars_expr` <- function(x, name) {
+  member_names <- ls(x, all.names = TRUE)
+  method_names <- names(polars_expr__methods)
+
+  if (name %in% member_names) {
+    env_get(x, name)
+  } else if (name %in% method_names) {
+    fn <- polars_expr__methods[[name]]
+    self <- x
+    environment(fn) <- environment()
+    fn
+  } else {
+    NextMethod()
+  }
+}
+
+#' @exportS3Method utils::.DollarNames
+.DollarNames.polars_expr <- function(x, pattern = "") {
+  member_names <- ls(x, all.names = TRUE)
+  method_names <- names(polars_expr__methods)
+
+  all_names <- union(member_names, method_names)
+  filtered_names <- findMatches(pattern, all_names)
```

Comment:
I've never used `findMatches()` but can you specify `fuzzy = FALSE`?

Reply:
Just to be sure, why is that?
Not sure which is more convenient.